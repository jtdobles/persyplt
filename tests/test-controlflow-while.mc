void main(){
	int a = 2;
    int b = 4;
	while(a > 1){
		prints('a is greater than 1');
		a = 1;
		// a = 3;
	}
	while(b == 4){
		prints('b is 4');
		b = 3;
	}
}