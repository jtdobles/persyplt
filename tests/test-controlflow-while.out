'a is greater than 1'
'b is 4'
